rbi/stdlib/big_math.rbi

```diff
@@ -92,7 +92,7 @@ module BigMath
     )
```

Comment:
I am thinking the type for `x` should actually be `BigDecimal`. I am not sure the best way to confirm this? 

https://ruby-doc.org/stdlib-2.5.1/libdoc/bigdecimal/rdoc/BigMath.html

Reply:
I think this should be, at a minimum, `T.any(Integer, BigDecimal)`.  I don't know whether the type should be wider than that (e.g. should it include `Float`), but given that you are the first person to notice that this RBI file was wrong in the first place, the minimum fix is fine.

I think you also want the same treatment for all the non-`prec` arguments in this RBI?